pkg/machine/qemu/machine.go

```diff
@@ -517,14 +514,22 @@ func (v *MachineVM) Start(name string, _ machine.StartOptions) error {
 	if err != nil {
 		return err
 	}
-
 	fd, err := qemuSocketConn.(*net.UnixConn).File()
 	if err != nil {
 		return err
 	}
 
+	dnr, err := os.Open("/dev/null")
```

Comment:
Os.Open() only opens with read access so you cannot use it for stdout/err file descriptors.

Reply:
@baude You want https://pkg.go.dev/os#OpenFile so you can specify how the file will be opened